server_tests/test/clientTests.js

```diff
@@ -32,7 +31,8 @@ describe("RUST API TEST SUITE", function() {
     var isBublcam = testModel === camModels.BUBLCAM1_0 || testModel === camModels.BUBLCAM1_2 || testModel === camModels.BUBLMOCK;
     var isMock = testModel === camModels.BUBLMOCK;
     var testViaWifi = process.env.SCARLET_TEST_WIFI === '1';
-    function shouldNotGetHere(err) {
+    var timeoutValue = testViaWifi? 45000 : 30000;
```

Comment:
But USB is significantly faster? Since the majority of the sporadic errors we've been encountering are timeout errors, wouldn't allowing more time help?